### PR TITLE
added: support extract and compile for same db types

### PR DESCRIPTION
**Pull request by nbesimi**

### CODE REVIEW

File: cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java [hunk 1]
```diff
@@ -141,14 +141,20 @@ private void compile(@CommandLine.Option(names = {"-s", "--source"}) String sour
...
-            Translator<Database, Database> translator = TranslatorFactory
-                    .translator(source.getDbType(), target.getDbType());
-
-            translatedModels = getDatabases(sourceWorkspace)
-                    .map(translateDatabases(translator))
-                    .collect(Collectors.toList());
-
-            translatedModels.forEach(writeOutput(targetWorkspace));
+            if (source.getDbType().equals(target.getDbType())) {
+                log.info("Skipping translation because the target ({}) and source ({}) db types are the same.",
```
Comment (nbesimi) [inline cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java:145]: Still checking if we need to add **model.yml** in the target DB, during compile where Source and Target are the same DB Type

Comment (nbesimi) [reply cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java:145]: Added the **model.yml** also

File: cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java [hunk 2]
```diff
@@ -141,13 +148,19 @@ private void compile(@CommandLine.Option(names = {"-s", "--source"}) String sour
...
             Files.createDirectory(targetWorkspace);
 
-            Translator<Database, Database> translator = TranslatorFactory
-                    .translator(source.getDbType(), target.getDbType());
+            if (source.getDbType().equals(target.getDbType())) {
+                log.info("Skipping translation because the target ({}) and source ({}) db types are the same.",
+                        target.getDbType(), source.getDbType());
+                translatedModels = getDatabases(sourceWorkspace).collect(Collectors.toList());
+            } else {
+                Translator<Database, Database> translator = TranslatorFactory
```
Comment (ailegion) [inline cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java:156]: this section looks like duplicate - can we put it on separate method? otherwise ltgm

**Review by ailegion — CHANGES_REQUESTED** [review]

**Review by ailegion — APPROVED** [review]
lgtm

**Pull request merged**